acl.go

```diff
@@ -304,32 +326,16 @@ func SetDefault(path string, acl ACL) error {
 	return setDefault(path, acl)
 }
 
-// TODO(joshlf): It seems as though the mask also
-// affects entries with the tag TagGroupObj, so
-// when calculating the new mask, its bits should
-// be taken into account as well.
-
-// Add adds the given entries to the ACL on path.
-// Any matching entries that exist on the file
-// will be overwritten. Two entries match if they
-// have the same tag (and, if that tag is TagUser
-// or TagGroup, they also have the same qualifier).
-//
-// In order to ensure that the new ACL is valid,
-// after being calculated from the old ACL and the
-// new entries, the new ACL is modified as follows:
-// If the ACL includes named user or group entries
-// (with the tags TagUser or TagGroup) but no mask
-// entry, a mask entry is added. This entry's
-// permissions are the union of all permissions
-// affected by the entry (namely, all entries with
-// the tags TagUser, TagGroup, or TagGroupObj).
-func Add(path string, entries ...Entry) error {
-	old, err := get(path)
-	if err != nil {
-		return err
+// SetDefault sets the default ACL on path,
+// returning any error encountered.
```

Comment:
Update this doc comment to reflect the behavior of this function rather than the behavior of `SetDefault`?